### ISSUE TITLE
Website and documentation down?

### ISSUE THREAD

**Issue by apkallum** [bitnine-oss/agensgraph#614]
Both [www.agensgraph.org](http://www.agensgraph.org/) and http://bitnine.net/documentations/quick-guide-1-3.html return 4xx errors.

**Comment by yjy44** [comment]
It has been confirmed that there is no problem with the connection.
Are you still having trouble connecting?

**Comment by apkallum** [comment]
Yes, I get 403 and redirected to: http://bitnine.net/?ckattempt=3

**Comment by uhayat** [comment]
Closing this ticket as website and documentation is alive. Please reopen if still face issue.
